source/ConditionSet.cpp

```diff
@@ -288,6 +288,11 @@ void ConditionSet::Add(const DataNode &node)
 	// non-simple operator (e.g. <=) and any number of simple operators.
 	static const string UNRECOGNIZED = "Warning: Unrecognized condition expression:";
 	static const string UNREPRESENTABLE = "Error: Unrepresentable condition value encountered:";
+
+	for(const string &token : node.Tokens())
+		if(token.size() > 0 && token[0] == '&')
+				node.PrintTrace("Warning: Only condition references should begin with \"&\". Unexpected \"&\" in (" + token + ").");
```

Comment:
could you name this CONDITION_PREFIX and declare it as a static const of Condition?

Reply:
This isn't a property of the Condition class; it should know nothing about how conditions are read or written. The DataNode reads the conditions and ConditionSet already knows about DataNode.h in its header. How does a DataNode static member sound?

Reply:
what matters to me mostly is this '&' is defined at one place so we can easily change it etc
sure datanode makes sense